filewriter_v2/README.md

```diff
@@ -0,0 +1,387 @@
+| [@image_number_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `image_number` field with a dimension of the data, in this case `0`.
+| [@start_time_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `start_time` field with a dimension of the data, in this case `0`.
+| [@channel_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `channel` field with a dimension of the data, in this case `1`.
+| [@signal](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-signal-attribute) | NX_CHAR | Declares which field is the default to be plotted, `"data"`. |
+| [image_number](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_NUMBER[] | Dimension scale defining the axis `image_number` of the data, typically this is `[1, ..., nP]`. |
+| [start_time](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_NUMBER[] | Dimension scale defining the axis `start_time` of the data. These are the relative start times for all images with absolute reference `/entry/start_time`. |
+| [channel](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_CHAR[][^2] | Dimension scale defining the axis `channel` of the data. These are the enabled channels, for example `["threshold_1", "threshold_2", "difference"]`. |
+| [data](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-data-data-field) | NX_NUMBER[nP,ch,i,j] | The image data.<br> **Note:** Files in V2 format always have only one `"/entry/data/data"` dataset, even in case `nimages_per_file` is greater than `0`. This is because it is a [Virtual Dataset](https://portal.hdfgroup.org/display/HDF5/Virtual+Dataset++-+VDS). |
+
+## /entry/instrument
+
+Collection of the components of the instrument or beamline.
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXinstrument"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-group) |
+| [name](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-name-field) | NX_CHAR | Name of the instrument. |
+| [time_zone](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-time-zone-field) | NX_DATE_TIME | ISO 8601 time_zone offset from UTC. |
+
+## /entry/instrument/beam
+
+Properties of the neutron or X-ray beam.
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXbeam"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-beam-group) |
```

Comment:
Link directly to https://manual.nexusformat.org/classes/base_classes/NXbeam.html#nxbeam